src/pages/FindPW/FindPW.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './FindPW.scss';
+
+const FindPW = () => {
+  return (
+    <div className="findPWFrame">
+      <h1 className="findpwText">비밀번호 찾기</h1>
+      <form className="formPWFrame">
+        <input
+          className="emailInput"
+          type="text"
+          placeholder="이메일을 입력해주세요"
+        ></input>
```

Comment:
content 가 없는 태그에 대해서는 self closing 신경써주세요
```suggestion
       />
```

Reply:
넵 똑같이 수정후 적용 완료했습니다.